src/app/services/variant-service.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable max-len */
+import { VariantAssignmentRequest } from 'expvariantassignmentsdk/src/interfaces/VariantAssignmentRequest';
+import {VariantAssignmentServiceClient} from 'expvariantassignmentsdk/src/contracts/VariantAssignmentServiceClient';
+import { VariantAssignmentClientSettings } from 'expvariantassignmentsdk/src/contracts/VariantAssignmentClientSettings';
+import { errorTypes, telemetry } from '../../telemetry';
+import { readFromLocalStorage, saveToLocalStorage } from '../utils/local-storage';
+import { EXP_URL } from './graph-constants';
+import { SeverityLevel } from '@microsoft/applicationinsights-web';
+
+
+interface TasResponse {
+  Id: string;
+  Parameters: Parameters;
+}
+interface Parameters {
+  [key: string]: string | boolean | number;
+}
+class VariantService {
+
+  static myInstance: VariantService = null as any;
```

Comment:
This variable appears unused. Do we need it?

Reply:
Ah, good catch. We don't need it